package.json

```diff
@@ -77,6 +79,7 @@
     "redlock": "^3.1.2",
     "thrift": "0.12.0",
     "ts-node": "8.0.2",
+    "tsd": "0.7.0",
```

Comment:
`tsd` is a [deprecated library](https://github.com/DefinitelyTyped/tsd).

Reply:
No, seems you mean this:
https://github.com/DefinitelyTyped/tsd

But used this:
https://github.com/SamVerschueren/tsd

I had seen, that @sindresorhus use it in https://github.com/sindresorhus/type-fest and think, that it good library for testing typescript declaration. Anyway, it better, that before with `hello-world` :)

Reply:
Ohh, sorry. Yes, I found the other.